src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/actions/PlaySettingsActions.kt

```diff
@@ -8,13 +8,15 @@ import com.jetbrains.rider.plugins.unreal.UnrealHost
 
 class PlaySettings : DefaultActionGroup() {
     override fun update(e: AnActionEvent) {
+        super.update(e)
         e.presentation.isVisible = e.getHost()?.isUnrealEngineSolution ?: false
         e.presentation.isEnabled = e.getHost()?.isConnectedToUnrealEditor ?: false
```

Comment:
I wonder if this should be visible and enabled for connected version only

Reply:
IMHO, it looks ok for now and we got used to current UX